Jenkinsfile

```diff
@@ -16,6 +16,7 @@ pipeline {
         stage('sphinx') {
           agent { label 'small' }
           steps {
+            sh 'rm -rf ./blacbox/.venv'
```

Comment:
You referred to a performance speed bump when wiping the whole virtual environment. What about invoking `pip uninstall crate-docs-theme`, to work around specific errors of staleness of this error?

Otherwise, is there a chance to run a later `pip install` with `--upgrade`?

Reply:
imho we should just wipe everything, as who knows, in the future might be another dependency that is failing.

Reply:
> Otherwise, is there a chance to run a later `pip install` with `--upgrade`, which might be missing there?

Ah, right, `bootstraph.sh`. It is using `--upgrade` already.
```
"$DIR/.venv/bin/pip" install --upgrade pip setuptools wheel
"$DIR/.venv/bin/pip" install --upgrade --requirement "$DIR/requirements.txt"
```

> imho we should just wipe everything

All right. Maximum idempotency, right. Downloaded packages will still be cached.